internal/handlers/pg/pgdb/query_test.go

```diff
@@ -399,14 +406,21 @@ func TestPrepareWhereClause(t *testing.T) {
 			)),
 			args:     []any{`v`, fmt.Sprint(float64(2 << 61))},
```

Comment:
```suggestion
			args:     []any{`v`, float64(2 << 61)},
```

Float values are no longer stored as strings

Reply:
This test checks for arguments passed to the PostgreSQL, which are passed as strings. The `42` value is passed as `"42"` and the `"foo"` value is passed as `"\"foo\""`

Reply:
… but they should not be passed as strings

Reply:
We should not call `MarshalSingleValue` in `prepareWhereClause`. Instead, we should provide a value as it is stored in PostgreSQL: underlying numerical type, string, etc.

Reply:
It's done by `pjson.MarshalSingleValue` function. From last investigations I highly believe that the function might be the reason of the bug of this PR so I will cover this